src/KeyVault/KeyVault/Commands/RemoveAzureKeyVaultAccessPolicy.cs

```diff
@@ -332,6 +332,10 @@ public override void ExecuteCmdlet()
                 {
                     if (string.IsNullOrWhiteSpace(this.ObjectId))
                     {
+                        if (ActiveDirectoryClient == null)
```

Comment:
Do we always need to throw if no active directory client is present, or is it possible that we only have to deal with object ids?